cylc/flow/network/graphql.py

```diff
@@ -80,6 +77,17 @@ def grow_tree(tree, path, leaves=None):
         tree_loc[len(path) % 2].update({'leaves': leaves})
 
 
+def execution_result_to_dict(execution_result):
```

Comment:
What it does is obvious, but a docstring could tell me:

* Why it does it
* What the  `execution_result` object is. A couple of examples might be nice?

Reply:
A couple of examples as a quick doctest would be a good idea, codecov is flagging this method as largely untested.

Reply:
As I mentioned in a different comment, we should be able to just use ExecutionResult.formatted instead of this.

See https://github.com/graphql-python/graphql-core/blob/b7935f75511dde280325631541f4a8a24b245709/src/graphql/execution/execute.py#L141-L149

Reply:
This has been removed, and the UIS side has been moved to `ExecutionResult.formatted`  (I believe the dictionary form is required as this is just before the payload is converted to JSON and sent).. Doing this at the `cylc-flow` end was an attempt to be consistent..